test/integration/008_schema_tests_test/models-v2/models/schema.yml

```diff
@@ -78,3 +78,12 @@ models:
           tests:
             - accepted_values: { values: ['red'] }
             - relationships: { from: favorite_color, to: ref('table_copy') }
+
+# all of these constraints will fail
+    - name: table_failure_null_relation
+      description: "A table with a null value where it should be a foreign key"
```

Comment:
`table_failure_null_relation` doesn't actually contain a null value though, does it? It looks like it just contains one row:
```
105 | count(*)
```

Reply:
that's correct. The target relation (`table_failure_copy`) has a `null` in that column, and there is no id `105` in `table_failure_copy`, so before this change the test incorrectly passed. Now the test fails as it should.

Reply:
oh, perfect!